### DISCUSSION TITLE
FLORIS v3 Calculate Power Output across wind direction with and without wake

### DISCUSSION THREAD

**Discussion by Nellytado** [NatLabRockies/floris#895] [Q&A] [answered]
Hello everyone,

I am currently working on a student project where I'm using Floris 3.6 to study the wake effect on wind farm power. Specifically, I need to analyze how power output varies with wind direction, both with and without the wake effect.

I've reviewed examples 4 through 6 provided in the documentation, but they don't seem to cover how to generate power output data for different wind directions without the wake. Could anyone guide me on how to approach this, or point me to any additional resources or examples that might help?

I would greatly appreciate any assistance or insights you can provide. Thank you for your support!

**Comment by bayc** [comment, marked as answer]
Hello @Nellytado ,

In v3.6, there is another function on `FlorisInterface` for this purpose: `calculate_no_wake` (https://github.com/NREL/floris/blob/v3.6/floris/tools/floris_interface.py#L159).  You can then use similar techniques like in example 4 to get the turbine powers by wind direction of the `fi.get_farm_power()` to get farm power per wind direction.

We would recommend you update to v4 when possible, as v3 will have limited support moving forward. Example 005 shows how to compute the no wake case (https://github.com/NREL/floris/blob/v4.0.1/examples/005_getting_power.py#L62).
